### ISSUE TITLE
Gaphor not starting on Windows machine

### ISSUE THREAD

**Issue by teunhoevenaars** [gaphor/gaphor#3450]
### Describe the bug
On one of my devices (see below for details) neither the installed version (Download Installer) nor the portable version starts up. A Gaphor shell is created and is being shown in the taskbar, but when I click it nothing happens, and hovering over the icon shows only an empty window in preview.

### To Reproduce
1. Start Gaphor on my Windows machine

### Expected behavior
Gaphor starts

### Screenshots
<img width="133" alt="bug_prntscrn" src="https://github.com/user-attachments/assets/415d8f38-f746-439c-b99f-9782b11c5748">

### OS
- [ ] Linux (Please put in notes the specific distro)
- [ ] macOS
- [x] Windows

### Version
Version of Gaphor: 2.26.0, but also 2.25.0

### Additional information
| OS Name              | Microsoft Windows 10 Home |
| -------------------- | ------------------------- |
| Version              | 10.0.19045 Build 19045    |
| Other OS Description | Not Available             |
| OS Manufacturer      | Microsoft Corporation     |
| System Manufacturer  | Acer                      |
| System Model         | Spin SP513-51             |
| System Type          | x64-based PC              |
| Processor                       | Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz, 2712 Mhz, 2 Core(s), 4 Logical Processor(s) |
| BIOS Version/Date               | Insyde Corp. V1.14, 02/03/2018                                                        |
| SMBIOS Version                  | 3                                                                                     |
| Embedded Controller Version     | 1.06                                                                                  |
| BIOS Mode                       | UEFI                                                                                  |
| BaseBoard Manufacturer          | KBL                                                                                   |
| BaseBoard Product               | Drift_SK                                                                              |
| BaseBoard Version               | V1.14                                                                                 |
| Platform Role                   | Mobile                                                                                |
| Secure Boot State               | On                                                                                    |
| Installed Physical Memory (RAM) | 8.00 GB

**Comment by danyeaw** [comment]
Hi @teunhoevenaars, are you running a virus scanner on this computer?

**Comment by teunhoevenaars** [comment]
hi @danyeaw , only the standard Windows 10 Virus & threat protection. Turning that Off did not have any effect. I also tried turning firewall Off, with equal lack of impact.

**Comment by amolenaar** [comment]
@teunhoevenaars Do you have any environment variables set that may interfere? Something on `PATH`, or `GI_TYPELIB_PATH`?

**Comment by teunhoevenaars** [comment]
hi @amolenaar apologies for the slow response, but yes: I see some `PATH` variables defined. Anything I should be looking for specifically?

**Comment by alvaro562003** [comment]
Hello There,
Having the same type of issue with Gaphor, i solved the problem by removing the directory :
C:\Users\xxxx\AppData\Local\Microsoft\Windows\INetCache\gaphor.

**Comment by teunhoevenaars** [comment]
I found out that my issue lies with my Windows machine. I tried another Windows PC and there it works fine. Also, I tried to install the example PyGObject as described on https://pygobject.gnome.org/getting_started.html#windows-logo-windows which results in similar errors when executing the `gtk4-demo` command.

As per https://gitlab.gnome.org/GNOME/gtk/-/issues/2201 I tried the options `GSK_RENDERER=cairo` flag, which does make `gtk4-demo` work! I also tried the `LIBGL_ALWAYS_SOFTWARE=true` flag, but that also gave no success.

As such, this seems to be a problem with the hardware of this Windows machine, and is expected behavior. Therefore, this bug can be closed AFAIK.
